src/D3_Arc.re

```diff
@@ -6,10 +6,10 @@ type t;
 
 [@bs.module "d3-shape"] external make : unit => t = "arc";
 
-[@bs.send] external innerRadius : t => float => t = "";
-[@bs.send] external outerRadius : t => float => t = "";
-[@bs.send] external cornerRadius : t => float => t = "";
-[@bs.send] external startAngle : t => float => t = "";
-[@bs.send] external endAngle : t => float => t = "";
-[@bs.send] external padAngle : t => float => t = "";
-[@bs.send] external centroid : t => 'a => (float, float) = "";
+[@bs.send] external innerRadius : t => float => t = "innerRadius";
```

Comment:
I haven't followed BuckleScript for a while, but when these files were initially written, using an empty string `""` name in bindings defaulting to using the same name as the external. Is that no longer supported?

Reply:
Throws a warning now so every time its present in your file, compilers lights up terminal output and your files with yellow squiggly lines.